### PR TITLE
Translation pt_umbrella-projects.md

### PR DESCRIPTION
**Pull request by getbrandao**

### CODE REVIEW

File: pt/lessons/advanced/umbrella-projects.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+- [IEx](#iex)
+
+## Introdução
+
+Para criar um projeto guarda-chuva nós iniciamos um projeto como se iniciássemos um projeto Mix normal, mas colocando o argumento `--umbrella`. Neste exemplo, faremos *the shell* de um kit de ferramentas de aprendizado de máquina. Por quê um kit de ferramentas de aprendizado de máquina? Por quê não? É composto de vários algoritmos de aprendizado e funções de utilidades diferentes.
+
+```shell
+$ mix new machine_learning_toolkit --umbrella
+
+* criando .gitignore
```
Comment (pjhampton) [inline pt/lessons/advanced/umbrella-projects.md]: Can you change the default language of Mix? I thought it was all in english.

**Comment by getbrandao** [comment]
@pjhampton I really don't know, I thought that the outputs used the user locale. If you want, I can change it! I'm sorry that!

**Comment by pjhampton** [comment]
First, don't be sorry! You're doing a kick-ass job!

Second, I might be wrong - I don't know if it does or doesn't. I'm asking you :-)

Do you know @pragmaticivan?

**Comment by pragmaticivan** [comment]
I didn't understand, are you talking about the mix lesson or the word mix in this file? Mix is the name of the project and we don't need to translate.

**Comment by pjhampton** [comment]
No I'm talking about the output from the console. I thought it only ever came in English?
```
~~> mix new blog --sup
* creating README.md
* creating .gitignore
* creating mix.exs
* creating config
* creating config/config.exs
* creating lib
* creating lib/blog.ex
* creating test
* creating test/test_helper.exs
* creating test/blog_test.exs

Your Mix project was created successfully.
You can use "mix" to compile it, test it, and more:

    cd blog
    mix test

Run "mix help" for more commands.
```

So creating is in english. Maybe Elixir (Mix) provides support for other languages?

**Comment by pragmaticivan** [comment]
I never got that in Portuguese hahaha even using bundle.

**Comment by pjhampton** [comment]
So @getbrandao when you run the command `mix new blog --sup` in your terminal, do you see the word `criando` or `creating`?

**Comment by pragmaticivan** [comment]
Ahhh I see now, it should not be translated. I never translate code, by the way.

**Comment by pjhampton** [comment]
Ahh right. Ok. In that case @getbrandao let's just keep the code / shell output the way it is in the english lessons :+1: 

I think we need to update the wiki just so everyone knows.

**Comment by getbrandao** [comment]
@pragmaticivan @pjhampton That's my problem, I don't have Portuguese language package. So, I translated thinking could be that way. Anyway, I will correct it!

**Comment by getbrandao** [comment]
I have another question... in the case the phrases/texts like:
``` Mix project was created successfully.
You can use "mix" to compile it, test it, and more: ```

Can be translated? 
Please!

**Comment by pjhampton** [comment]
@getbrandao No I don't think so. Anything words / sentences / phrases / paragraphs that Elixir or Erlang generate, leave it :-)

Just translate human content that us lesson designers have been creating!

**Comment by getbrandao** [comment]
Thank you all! I made the corrections!

I look forward to hearing from you! :-)

File: pt/lessons/advanced/umbrella-projects.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
+---
+layout: page
+title: Umbrella Projects
```
Comment (pragmaticivan) [inline pt/lessons/advanced/umbrella-projects.md]: Translate the title, "Projetos Guarda-chuva" or "Projetos Umbrella"

File: pt/lessons/advanced/umbrella-projects.md [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+
+Determinados momentos um projeto pode ficar enorme, realmente enorme. A ferramenta de construção Mix nos permite dividir nosso código em vários aplicativos e fazer nossos projetos em Elixir mais manejáveis à medida que crescem. 
+
+## Sumário
+
+- [Introdução](#introducao)
+- [Projetos filho](#projetos-filho)
+- [IEx](#iex)
+
+## Introdução
```
Comment (pragmaticivan) [inline pt/lessons/advanced/umbrella-projects.md]: You should use `a name` here : `<a name="introducao">`

File: pt/lessons/advanced/umbrella-projects.md [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+
+Para criar um projeto guarda-chuva nós iniciamos um projeto como se iniciássemos um projeto Mix normal, mas colocando o argumento `--umbrella`. Neste exemplo, faremos *the shell* de um kit de ferramentas de aprendizado de máquina. Por quê um kit de ferramentas de aprendizado de máquina? Por quê não? É composto de vários algoritmos de aprendizado e funções de utilidades diferentes.
```
Comment (pragmaticivan) [inline pt/lessons/advanced/umbrella-projects.md]: `faremos o *shell*`

Comment (pragmaticivan) [inline pt/lessons/advanced/umbrella-projects.md]: `Por que` in beginning of questions doesn't use accentuation.

**Comment by pragmaticivan** [comment]
@getbrandao just left some observations. Thanks! o/ :beers:

Happy St Patrick's day!
wishing-you-a-very-happy-st-patricks-day-glitter

File: pt/lessons/advanced/umbrella-projects.md [hunk 5]
```diff
@@ -10,13 +10,13 @@ Determinados momentos um projeto pode ficar enorme, realmente enorme. A ferramen
 
 ## Sumário
 
-- [Introdução](#introducao)
+- [Introdução](#introdução)
```
Comment (pragmaticivan) [inline pt/lessons/advanced/umbrella-projects.md]: You should not use accentuation for anchor links. Use `#introducao` and the link on title. 

You can check the `pipe-operator.md` in pt-BR

**Comment by getbrandao** [comment]
Guys, I would like to ask to someone make a revision in the commit. Thanks.

**Comment by pragmaticivan** [comment]
Solid @getbrandao can you squash the commits? Thanks! Almost there!

**Comment by getbrandao** [comment]
@pragmaticivan done!

**Comment by pragmaticivan** [comment]
@getbrandao nice you got the rebase skills!! o/ Thanks!

**Pull request merged**